src/components/nodes/textAnnotation/textAnnotation.vue

```diff
@@ -68,7 +67,7 @@ export default {
         newHeight = currentBoundsHeight;
       }
 
-      this.shape.resize(this.nodeWidth, newHeight - highlightPadding);
+      this.shape.resize(this.nodeWidth, newHeight - 3);
```

Comment:
This change was needed to allow the crown config mixing to be removed. I'll try to find a better place to declare this as a constant.